pkg/actions/fargate/tasks.go

```diff
@@ -19,16 +19,12 @@ type createFargateStackTask struct {
 
 func (t *createFargateStackTask) Describe() string { return "create fargate IAM stacK" }
 
-func makeClusterStackName(clusterName string) string {
-	return "eksctl-" + clusterName + "-fargate"
-}
-
 func (t *createFargateStackTask) Do(errs chan error) error {
 	rs := builder.NewFargateResourceSet(t.cfg)
 	if err := rs.AddAllResources(); err != nil {
 		return errors.Wrap(err, "couldn't add all resources to fargate resource set")
 	}
-	return t.stackManager.CreateStack(context.TODO(), makeClusterStackName(t.cfg.Metadata.Name), rs, nil, nil, errs)
+	return t.stackManager.CreateStack(context.TODO(), t.cfg.MakeStackName("fargate"), rs, nil, nil, errs)
```

Comment:
```suggestion
	"fargate"
```

This should probably be a constant somewhere that is accessible. Like in the `api` package.

Reply:
If I'm moving this to a constant in `api` then I would rather be moving all constant stack suffixes into there, is there anything against just having the naming funcs be in that package too?